config/routes.rb

```diff
@@ -312,6 +312,14 @@
   # static pages routes
   get 'pages/*id' => 'pages#show', constraints: { format: :html, id: Regexp.union(Gemcutter::PAGES) }, as: :page
 
+  scope :policies, to: 'pages#policies', constraints: { format: :html } do
+    get '', id: 'policies', as: 'policies'
+    get 'acceptable-use', id: 'acceptable_use'
+    get 'copyright-policy', id: 'copyright_policy'
+    get 'privacy-notice', id: 'privacy_notice'
+    get 'terms-of-service', id: 'terms_of_service'
+  end
+
```

Comment:
the get with the empty resource name is tad unclear, let's give it a proper name.

```suggestion
  get 'policies', id: 'policies', as: 'policies', to: "pages#policies"
  scope :policies, to: 'pages#policies', constraints: { format: :html } do
    get 'acceptable-use', id: 'acceptable_use'
    get 'copyright-policy', id: 'copyright_policy'
    get 'privacy-notice', id: 'privacy_notice'
    get 'terms-of-service', id: 'terms_of_service'
  end
```